Content.Server/GameObjects/Components/Atmos/AirtightComponent.cs

```diff
@@ -161,6 +161,8 @@ private void UpdatePosition()
         {
             if (Owner.Transform.Anchored)
             {
+                if (Owner.Transform.GridID == GridId.Invalid)
+                    return;
```

Comment:
@Zumorica

Reply:
Sure, change it. What's one more conflict to fix on my piping PR anyway :pensive: 
Also use `GridID.IsValid()`, don't check for equality with invalid.